lib/rules/template-vars.js

```diff
@@ -0,0 +1,66 @@
+const { getTemplateLocals } = require('@glimmer/syntax');
+
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description:
+        'require that variables used in <template> tags do not trigger no-unused-vars failures, and should trigger no-undef if they are not defined properly',
```

Comment:
Would it be more straightforward to just say:

```
require that variables used in <template> tags are defined before use and not unused
```

Reply:
Yup.  😅 Thanks, will update